app/flask_config.py

```diff
@@ -0,0 +1,21 @@
+"""Flask configuration class."""
+import os
+
+
+class Config:
+    def __init__(self):
+        self._boardId = os.environ.get('boardId')
+        self._key = os.environ.get('key')
+        self._token = os.environ.get('token')
+
+    @property
+    def boardId(self) -> str:
```

Comment:
Python variable and function names are conventionally snake_case

Reply:
Done - ceebcbaa676c59ee4d6ba0cced1af87366115635